wallet/udb/addressdb.go

```diff
@@ -646,6 +656,73 @@ func bip0044AccountInfo(pubKeyEnc, privKeyEnc []byte, nextExtIndex, nextIntIndex
 	return row
 }
 
+// deserializeVotingAccountRow deserializes the raw data from the passed
+// account row as a voting account.
+func deserializeVotingAccountRow(row *dbAccountRow) (*dbVotingAccountRow, error) {
```

Comment:
this func must check the length of the row data to ensure it doesn't panic due to OOB.  See deserializeBIP0044AccountRow for an example